.github/workflows/links.yml

```diff
@@ -0,0 +1,24 @@
+name: Links
+
+on:
+  workflow_dispatch:
+  pull_request:
+  push:
+    branches:
+      - master
+
+jobs:
+  linkChecker:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
+        with:
+          fetch-depth: 0
+
+      - name: Link Checker
+        uses: lycheeverse/lychee-action@f796c8b7d468feb9b8c0a46da3fac0af6874d374 #v2.2.0
+        with:
+          args: --no-progress --offline './content/en/**/*.md'
+          token: ${{ secrets.CUSTOM_TOKEN }}
```

Comment:
What is this? I couldn't find it set. 
Also, do we really need it? Is it safe to use this token on PRs?